aexpect/remote.py

```diff
@@ -309,7 +317,7 @@ def remote_login(client, host, port, username, password, prompt, linesep="\n",
     if host and host.lower().startswith("fe80"):
         if not interface:
             raise LoginError("When using ipv6 linklocal an interface must "
-                             "be assigned")
+                             "be assigned", None)
```

Comment:
Well the second argument is output, therefore '' would fit better here. Also I'd suggest making the argument optional (defaults to '') in the LoginError, rather than supplying empty output when raising the exc.

Reply:
Sure, resolved with the next push.